.github/workflows/deps_lint.yml

```diff
@@ -97,7 +97,10 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Build the Docker images
-      run: docker-compose build
+      run: |
+        sed -i 's/localhost:5001/gh_actions_host:3214/g' ./tests/test_config.json
```

Comment:
While this is a working solution, it should be set up with a custom Docker `config.json` file.
Either create one for CI in the `.docker` or `.ci` folder. Make a line in the `Dockerfile` that copies the file into the root folder (similar to how it copies over `./.docker/run.sh`), and finally update both the `Dockerfile` and the `docker-compose.yml` file with the "final" destination, e.g., `CONFIG_FILE=docker_config.json` or similar.

Reply:
I did this initially, but I guess it depends how we view the docker images. Are they just for CI or should we use them more generally? As they end up using the `test_config.json` anyway you're probably right that we should have another config file for them.

Reply:
A user who want to use this via Docker are instructed (or should be anyway) to change the `Dockerfile` (and/or possibly the `docker-compose.yml`) to their setup. This has always been the case, in my opinion.